integration/pvtdata/pvtdata_test.go

```diff
@@ -946,7 +946,7 @@ func initThreeOrgsSetup(removePeer1 bool) *nwo.Network {
 	client, err := docker.NewClientFromEnv()
 	Expect(err).NotTo(HaveOccurred())
 
-	config := nwo.FullSolo()
+	config := nwo.FullEtcdRaft()
```

Comment:
it seems there is mutation of the config afterwards, so why not just use the `MinimalRaft()` ?

Reply:
Thanks for the review @yacovm, it's definitely a bit awkward but I was trying to minimise the changes- since it gets mutated in the private data tests anyway, do you think it would be better to leave the shared networks alone and just handle the private data specific network config entirely in `pvtdata_test.go`?

Reply:
Honestly I think that since these are ginkgo integration tests, and they're not used by consumers, it's easiest to just leave them be, and when we remove Solo, we'll clean up this place as well since we won't have a choice.